src/dynamo0p3.py

```diff
@@ -1188,6 +1200,17 @@ def __init__(self, schedule):
         # We create a dictionary whose keys are the map names and entries
         # are the corresponding field objects.
         self._unique_fs_maps = {}
+        # We also create a dictionary of column-banded dofmaps. Entries
+        # in this one are a list where the first item is the
+        # corresponding CMA object and the second is whether the map
+        # corresponds to the 'to' or 'from' space.
+        self._unique_cbanded_maps = {}
```

Comment:
I agree. I felt slightly uncomfortable when I did that but didn't pause to examine why. I like your suggestion so have changed it. 475e151.